### PR TITLE
global styles

### PR DESCRIPTION
**Pull request by francoo-trainer**
se hicieron cambios en main.css, se agregaron los estilos globales.

### CODE REVIEW

**Comment by Jasoneto** [comment]
Hi, please add just two fonts. Great job.

**Review by Jasoneto — COMMENTED** [review]
Hi, please  add only the required fonts.

**Review by Brayan1425 — CHANGES_REQUESTED** [review]
please add only the requeried fonts

File: main.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/* fuentes externas*/
+@font-face {
+  font-family: "NotoSerif";
+  src: url(/assets/tipografia/NotoSerif-Regular.ttf);
+  font-display: swap;
+}
```
Comment (johanmendezb) [inline main.css]: Please add a line break so it aligns with the code style.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/* fuentes externas*/
```
Comment (johanmendezb) [inline main.css]: Please add comments in english.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  left: 35px;
+  top: 133px;
+
+  line-height: 110%;
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.01em;
+}
```
Comment (johanmendezb) [inline main.css]:
```suggestion
.heading__subtitle {
   position: absolute;
  align-items: center;
  display: flex;
  height: 40px;
  left: 159px;
  letter-spacing: 0.01em;
  line-height: 110%;
  text-align: center;
  top: 70px;
  width: 410px;
}
```

File: main.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+.content__text {
+  position: absolute;
+  width: 233px;
+  height: 186px;
+  left: 35px;
+  top: 180px;
+
+  line-height: 110%;
+
```
Comment (johanmendezb) [inline main.css]: `line-height` property should not be in between line breaks

File: main.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  height: 18px;
+  left: 341px;
+  top: 348px;
+
+  line-height: 110%;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.01em;
+
```
Comment (johanmendezb) [inline main.css]: Same as above.

**Review by johanmendezb — CHANGES_REQUESTED** [review]
Please review the comments.
Also remove extra line breaks and organize the css properties alphabetically.

File: main.css (hunk 1 above)
Comment (francoo-trainer) [reply main.css]: thanks for the observation, already working on it

File: main.css (hunk 2 above)
Comment (francoo-trainer) [reply main.css]: thanks for the observation, already working on it

**Review by Theweeksad — APPROVED** [review]
good job, you changed the fonts

File: css/global-style.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  font-family: "NotoSerif";
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 39.6px;
+}
+
+h2 {
+  font-family: "Nunito";
+  font-size: 24px;
+  font-size: 24px;
```
Comment (Jasoneto) [inline css/global-style.css]: You've already write the same font-size variable in line 19.

**Review by Jasoneto — APPROVED** [review]
this PR has achieved with the changes suggested.

**Review by johanmendezb — APPROVED** [review]

**Review by Theweeksad — APPROVED** [review]
All ready!!!

**Review by Brayan1425 — APPROVED** [review]
Changes are good

**Review by Theweeksad — APPROVED** [review]
good job

**Pull request merged**